tests/test_validation.py

```diff
@@ -73,6 +73,65 @@ def setup_1bin_shapesys(
     }
 
 
+@pytest.fixture(scope='module')
+def spec_1bin_lumi(source=source_1bin_example1()):
+    spec = {
+        'channels': [
+            {
+                'name': 'singlechannel',
+                'samples': [
+                    {
+                        'name': 'signal',
+                        'data': source['bindata']['sig'],
+                        'modifiers': [
+                            {'name': 'mu', 'type': 'normfactor', 'data': None}
+                        ],
+                    },
+                    {
+                        'name': 'background',
+                        'data': source['bindata']['bkg'],
+                        'modifiers': [{'name': 'lumi', 'type': 'lumi', 'data': None}],
+                    },
+                ],
+            }
+        ]
+    }
+    return spec
+
+
+@pytest.fixture(scope='module')
+def expected_result_1bin_lumi(mu=1.0):
+    ## TO DO - FIX THIS
```

Comment:
what's to fix here?

Reply:
I need to run this with HistFactory to get actual expected results.